sklearn/utils/_estimator_html_repr.css

```diff
@@ -14,16 +14,16 @@
   --sklearn-color-fitted-level-3: cornflowerblue;
 
   /* Specific color for light theme */
-  --sklearn-color-text-on-default-background: var(--theme-code-foreground, var(--jp-content-font-color1, black));
-  --sklearn-color-background: var(--theme-background, var(--jp-layout-color0, white));
-  --sklearn-color-border-box: var(--theme-code-foreground, var(--jp-content-font-color1, black));
+  --sklearn-color-text-on-default-background: var(--sg-text-color, var(--theme-code-foreground, var(--jp-content-font-color1, black)));
```

Comment:
Just to double check that I understand this CSS (in which case TIL CSS vars!): this is going to set `--sklearn-color-text-on-default-background` to the value of `--sg-text-color` if it exists, falling back to `--theme-code-foreground`, falling back to `--jp-content-font-color1` and if none of those exists `black`?

Reply:
yes exactly. So checking the rendering, it seems that does not solve the issue because sphinx-gallery is detecting that I am in a dark mode while we don't have dark mode in scikit-learn. Maybe it uses the OS then.

Reply:
So sphinx-gallery sets the value of `--sg-text-color` based on the OS setting?

Reply:
So to make it work, I need to add `data-theme="light"` in the `<html>` tag since this is the selector imposed by sphinx-gallery.